script/Arcueid/Day 9/09_00_ARC09_1.txt

```diff
@@ -136,8 +136,8 @@ A child lies crumpled in the clearing.
 [sha:9b13c7cf5b90dbcd5fdd4990fe9f41bdac9d2939]{
 -- Page 5, Offset 21319.
 -- 白いシャツが真っ赤に染まって、ぴくりとも動かない。
-A deep red stain has blossomed on their white shirt.
- They don't move so much as an eyebrow.
+A deep red stain has blossomed on his white shirt.
+ He doesn't move so much as an eyebrow.
```

Comment:
I'd intentionally kept this impersonal, under the impression that SHIKI here feels very dispassionate towards 'the child'. He is simply killing those he can kill, bothering to appreciate details about them doesn't seem like it would be in character. I'd initially even had this as `it`, for a more potent dehumanization. Maybe get another opinion as well.

Reply:
Sounds reasonable but here Shiki was actually looking at himself, iirc it was that SHIKI stabbed Shiki which then Makihisa came and almost injured SHIKI to death, the only reason Shiki was remembering the scene from SHIKI's perspective is because of thier souls being linked.

Reply:
I'm for keeping it as their. Don't feel strongly about it though